pennylane/operation.py

```diff
@@ -464,10 +464,9 @@ def parameters(self):
         """Current parameter values."""
         return self.data.copy()
 
-    def queue(self):
+    def queue(self, context=qml.QueuingContext):
```

Comment:
Only minor, but I always thought it is cleaner to use `None` and then instantiate in the function?

Reply:
I did that originally, but realized that no instantiation is happening here --- `qml.QueuingContext` is simply passing an uninstantiated class. So this should be as safe as passing `None`, while being slightly more elegant